cms/static/js/views/paging.js

```diff
@@ -0,0 +1,143 @@
+;(function (define) {
```

Comment:
I don't understand what you're asking about preserving history. Are you requesting that this be a revert commit?